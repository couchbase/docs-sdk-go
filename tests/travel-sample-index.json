{
  "name": "travel-sample-index",
  "type": "fulltext-index",
  "params": {
   "doc_config": {
    "docid_prefix_delim": "",
    "docid_regexp": "",
    "mode": "scope.collection.type_field",
    "type_field": "type"
   },
   "mapping": {
    "default_analyzer": "standard",
    "default_datetime_parser": "dateTimeOptional",
    "default_field": "_all",
    "default_mapping": {
     "dynamic": true,
     "enabled": false
    },
    "default_type": "_default",
    "docvalues_dynamic": false,
    "index_dynamic": true,
    "store_dynamic": false,
    "type_field": "_type",
    "types": {
     "inventory.hotel": {
      "dynamic": false,
      "enabled": true,
      "properties": {
       "description": {
        "enabled": true,
        "dynamic": false,
        "fields": [
         {
          "docvalues": true,
          "include_in_all": true,
          "include_term_vectors": true,
          "index": true,
          "name": "description",
          "store": true,
          "type": "text"
         }
        ]
       }
      }
     }
    }
   },
   "store": {
    "indexType": "scorch",
    "segmentVersion": 15
   }
  },
  "sourceType": "gocbcore",
  "sourceName": "travel-sample",
  "sourceParams": {},
  "planParams": {
   "maxPartitionsPerPIndex": 1024,
   "indexPartitions": 1,
   "numReplicas": 0
  },
  "uuid": ""
 }
