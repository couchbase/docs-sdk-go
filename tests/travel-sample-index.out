{"status":"ok","uuid":"5efd674cb704da51"}
